### DISCUSSION TITLE
success story by using simdjson

### DISCUSSION THREAD

**Discussion by lano1106** [simdjson/simdjson#2156] [General]
https://twitter.com/lano1106/status/1771222617594171855

simdjson is absolutely amazing!

**Comment by jkeiser** [comment]
Oh wow, that's amazing! And microsecond trading seems like a perfect application. Are the exchanges actually running on json APIs?

**Reply by lano1106** [reply]
most of them are JSON based APIs...

just visit the API doc of the top 10 of them at:
https://coinmarketcap.com/rankings/exchanges/

if you are looking for sponsors, I would think that this could be a good place to look...

**Reply by jkeiser** [reply]
Ah, I guess crypto exchanges aren't as microsecond-optimized as the stock exchanges. But I guess it means users have more room to eke out an advantage, like with simdjson :)
